src/main/java/seedu/anilist/logic/commands/Action.java

```diff
@@ -17,6 +17,8 @@
      */
     public static boolean isValidAction(String test) {
         switch (test) {
+        case "a":
+        case "d":
         case "add" :
         case "delete" :
             return true;
```

Comment:
Is this function updated for list?

Reply:
Havnt add the list action word yet, thinking of doing it in another branch